trimmer_decorator.rb

```diff
@@ -0,0 +1,5 @@
+class TrimmerDecorator < Decorator
```

Comment:
- [ ] Same here, Kindly import the `Decorator` file, like `require_relative 'decorator'`